bindings/matrix-sdk-ffi/src/calls.rs

```diff
@@ -0,0 +1,31 @@
+use matrix_sdk::RoomCall as SdkRoomCall;
+
+#[derive(uniffi::Object)]
+pub struct RoomCall {
+    pub(crate) inner: SdkRoomCall,
+}
```

Comment:
Currently, I only treat room calls. This is also the case in the newer (upcoming) PR. Since we don't really use/support any other types of calls, I did not implement it (as there would be no sensible way to test or use it and having a dead code does not look particularly good).